spacy_lookups_data/tests/test_lb.py

```diff
@@ -0,0 +1,28 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from spacy.lang.lb import Luxembourgish
+import pytest
+
+
+@pytest.fixture(scope="session")
+def lb_nlp():
+    return Luxembourgish()
+
+
+@pytest.mark.parametrize(
```

Comment:
I guess we should add `@pytest.mark.xfail` for now. (There's obviously no `spacy.lang.lb` yet, so this is currently expected to fail.)

Reply:
I changed it to `@pytest.mark.xfail`, but I am not sure, whether it works.